src/Application.vala

```diff
@@ -104,19 +104,21 @@ public class Terminal.Application : Gtk.Application {
                         if (!terminal.is_init_complete ()) {
                             terminal.set_init_complete ();
                         } else {
+                            var process_string = _("Process exited successfully");
+                            var process_icon = new ThemedIcon ("process-completed-symbolic");
+                            if (exit_status != 0) {
+                                process_string = _("Process exited unsuccessfully");
```

Comment:
I think technically the process exited/completed but _also_ had errors. I know it's kinda nit-picking but this feels more accurate.

```suggestion
                                process_string = _("Process exited with errors");
```